src/services/layer-metadata/layer-metadata.service.ts

```diff
@@ -19,12 +18,11 @@ export class LayerMetadataService {
   private localMetadataBaseUrl = 'https://map.geoportail.lu/getMetadata'
 
   async getLayerMetadata(
-    node: LayerTreeNodeModel,
+    id: string | number,
```

Comment:
Can't we use `LayerId` from '@/stores/map.store.model' instead?

Reply:
I was hesitating with this as it means imports of `LayerId` in many parts of the app (as `LayerTreeNodeModel` should actually also be typed the same to be consistent). But I think you are right as it improves typing. e0ac270ce17366d44960d3325f0c1657a4bdf6e3 might deserve a second look, thanks!